model/gen-ai/registry.yaml

```diff
@@ -177,6 +177,71 @@ groups:
               brief: 'Output tokens (completion, response, etc.)'
         brief: The type of token being counted.
         examples: ['input', 'output']
+      - id: gen_ai.agent.id   # alternatives: assistant (openai)
+        stability: experimental
+        type: string
+        brief: The unique identifier of the GenAI agent.
+        examples: ['asst_5j66UpCpwteGg4YSxUnt7lPY']
+      - id: gen_ai.agent.name
+        stability: experimental
+        type: string
+        brief: Human-readable name of the GenAI agent provided by the application.
+        examples: ["Math Tutor", "Fiction Writer"]
+      - id: gen_ai.agent.description
+        stability: experimental
+        type: string
+        brief: Free-form description of the GenAI agent provided by the application.
+        examples: ["Helps with math problems", "Generates fiction stories"]
+      - id: gen_ai.agent.tools
```

Comment:
those would be reported as part of tool definition on the agent creation, right?

A couple of thoughts:
1. names alone are probably not enough and we'd need to add other info, which, at that point might be too verbose. Should we do this as event instead? This way we capture all the details, but people can opt in/out of them.
2. I don't think we should put tools into the `agent` namespace - we should define attributes that are used without agents too